fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -50,6 +52,56 @@
 #define AES_256_KEY_LENGTH 32
 #define AES_256_IV_LENGTH 16
 
+class BlobCipherMetrics : public NonCopyable {
+public:
+	static BlobCipherMetrics* getInstance() {
+		static BlobCipherMetrics* instance = nullptr;
+		if (instance == nullptr) {
+			instance = new BlobCipherMetrics;
+		}
+		return instance;
+	}
+
+	// Order of this enum has to match initializer of counterSets.
+	enum UsageType : int {
```

Comment:
nit: `UsageType` -> `Type` ; given enum is defined under `BlobCipherMetrics`, `Type` seems sufficient (inferred as MetricsType)

Reply:
I'm feeling `Type` is more ambiguous on what it is used for, if we only look at the enum type name, and not the enum values in it.